drafts/one-percent-for-open-source.rst

```diff
@@ -0,0 +1,111 @@
+Many people doing this work are burning out,
+giving up,
+or just never standing in the first place.
+We need to improve the sustainability of open source in order to protect the valuable ecosystem that we have built,
+and help it grow in the future.
+
+Funding Open Source
+-------------------
+
+I'd like to propose an idea:
+
+    A program called **1% for open source**: A program where Venture Capitalist's and startup companies agree to invest 1% of their funding round into the open source software that they use. 
+
+I believe that this is a win/win/win for all involved:
+
+* **Venture Capitalists** invest in the open source ecosystem that makes companies *much cheaper to build*. It's also *really good marketing* for the VC firms, to show they are investing in the common tools we all rely on.
+* **Startups** invest in the tools that they use, to *ensure continued support and maintenance*, to remove risk from their future, and have better tools to build their company with. It will also make *hiring developers* much easier for the companies, because developers love companies that invest in open source.
+* **Open source projects** are able to *fund project management and maintenance* that is required to make them sustainable for the long term. This continues the *force multiplier* that open source has given to the software industry as a whole.
+
+Why Venture Capital payments?
+-----------------------------
+
+The biggest reason is because all parties involved benefit from open source.
+Secondly,
+having a concrete point in time also makes it much easier to define when the money should be invested.
+Third,
+a set amount of money is already changing hands during an investment,
+so having 1% that money to go open source is easy.
+
+I also think monetary investment is required to make a real dent in this problem. 
+A lot of places have things like "Open Source Fridays" and "20% time",
+but those programs are the first to be cut when a company has issues.
+Investing in open source as a lump sum during funding rounds makes sure the investment up front,
+where it can't be removed as an ongoing program.
+
+How should we implement it?
+---------------------------
+
+The startup who has gotten the investment should invest 1% of the money from the VC firm into the open source projects that they use.
+
+There are many different ways this could be implemented as a lower level.
+A few of my ideas are:
+
+* Startup companies create a list of the OSS projects they use, using tools like http://libraries.io/, and then give money to the 10 largest projects that have a way to accept money. This could be done during the license compliance phase of the investment process.
+* There could be a new non-profit foundation created, which gets all the donations and then has established criteria for how to reinvest the money as grants.
+* You could give the money to the existing foundations that focus on the area of open source that is most valuable to you 
```

Comment:
There's a bigger problem here that this essay doesn't address - We don't have the structures in place to make this happen. For example, the PSF doesn't have a great history of dispensing cash to make development happen. There are some notable exceptions - usually in the form of short term grants, but "ongoing payments" isn't something they've historically done. But they're also one of only a handful of organisations that are in a position to *accept* money in the Python ecosystem. If every valley VC firm did this *today*, a handful of foundations would get a windfall, and then scramble to work out what they would do. It's not enough to just say that we need to *get* money - we have to provide a place for that money to go, *and* have a clear strategy in place for how the money would be used.

Reply:
I'm proposing the structure in some ways, with the new non-profit. Most of the work would be figuring out exactly what you're talking about.